### PR TITLE
A user is prevented of doing something if is log out

### PR DESCRIPTION
**Pull request by anatoliikryvishyn**

### CODE REVIEW

**Comment by anatoliikryvishyn** [comment]
@serhiir, please test [here](https://trouble-tada-interlink.herokuapp.com) and review.

**Comment by serhiir** [comment]
@anatoliikryvishyn LGTM :+1: Please follow a few suggestions though.

File: app/assets/javascripts/session_logout.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    };
+
+    bindLogoutListener = function() {
+        window.addEventListener('storage', storageChange);
+    };
+
+    bindLogoutEvent = function() {
+        logoutLink.click(function() {
+            makeLogoutRequest(logoutLink.attr('href'));
+        });
```
Comment (serhiir) [inline app/assets/javascripts/session_logout.js]: @anatoliikryvishyn Can you introduce a local variable for url here please?

File: app/assets/javascripts/session_logout.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    };
+
+    makeLogoutRequest = function(url) {
+        $.ajax({
+            url: url,
+            method: 'GET',
+            success: function() {
+                window.localStorage.setItem('logout', true);
+            }
```
Comment (serhiir) [inline app/assets/javascripts/session_logout.js]: @anatoliikryvishyn The suggestion is to introduce a method, similar as for `clearLogoutStorage`.

File: app/assets/javascripts/session_logout.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        });
+    };
+
+    storageChange = function(event) {
+        if (event.key == 'logout') {
+            location.reload();
+        }
```
Comment (serhiir) [inline app/assets/javascripts/session_logout.js]: @anatoliikryvishyn I would name `onStorageChange` to outline it is been firing as the even occurs.

**Comment by serhiir** [comment]
@anatoliikryvishyn Done.

**Comment by anatoliikryvishyn** [comment]
@serhiir, updated.

**Pull request merged**